src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SingleSpaceSeparatorCheck.java

```diff
@@ -164,38 +164,52 @@ public boolean isCommentNodesRequired() {
 
     @Override
     public void beginTree(DetailAST rootAST) {
-        if (rootAST != null) {
-            visitEachToken(rootAST);
-        }
+        visitEachToken(rootAST);
     }
 
     /**
-     * Examines every sibling and child of {@code node} for violations.
+     * Examines every sibling and descendant of {@code node} for violations.
```

Comment:
When you renamed `node` to `rootAst`, you forgot to rename it here in the javadoc.